### PR TITLE
[FEATURE] Add stream_context option to WebOpcacheResetExecuteTask

### PR DESCRIPTION
**Pull request by sabbelasichon**
- This is useful for example to execute the task if we have to provide Basic Authentication

### CODE REVIEW

File: src/Task/Php/WebOpcacheResetExecuteTask.php [hunk 1]
```diff
@@ -35,10 +35,15 @@ public function execute(Node $node, Application $application, Deployment $deploy
             throw new \TYPO3\Surf\Exception\InvalidConfigurationException('No "scriptIdentifier" option provided for WebOpcacheResetExecuteTask, make sure to execute "TYPO3\\Surf\\Task\\Php\\WebOpcacheResetCreateScriptTask" before this task or pass one explicitly', 1421932610);
         }
 
+        $context = null;
+        if (isset($options['context']) && is_array($options['context'])) {
```
Comment (helhum) [inline src/Task/Php/WebOpcacheResetExecuteTask.php]: can you please call that `stream_context`

Comment (sabbelasichon) [inline src/Task/Php/WebOpcacheResetExecuteTask.php]: Sure. Only the option? Or the variable too?

**Comment by simonschaufi** [comment]
Fixes: #83

File: src/Task/Php/WebOpcacheResetExecuteTask.php (hunk 1 above)
Comment (simonschaufi) [inline src/Task/Php/WebOpcacheResetExecuteTask.php]: update?

**Comment by sabbelasichon** [comment]
I have fixed it. Hope everything is fine now.

**Comment by simonschaufi** [comment]
great. Maybe some documentation how to use it?

**Comment by sabbelasichon** [comment]
You can now just pass the option stream_context to the task TYPO3\\Surf\\Task\\Php\\WebOpcacheResetExecuteTask:

$workflow->setTaskOptions('TYPO3\\Surf\\Task\\Php\\WebOpcacheResetExecuteTask', array(
        'baseUrl' => $baseUrl,
        'context' => array(
            'http' => array(
                'header' => 'Authorization: Basic '.base64_encode("username:password"),
            ),
        ),
    ));

**Comment by simonschaufi** [comment]
@helhum can this be merged?

**Comment by simonschaufi** [comment]
@helhum can this be merged?

**Pull request merged**